eq-author-api/docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - JAEGER_SAMPLER_MANAGER_HOST_PORT=http://jaeger:5778/sampling
       - JAEGER_SAMPLER_TYPE=probabilistic
       - JAEGER_SAMPLER_PARAM=1
+      - GOOGLE_APPLICATION_CREDENTIALS=/app/.firebase-admin-service-account.json
```

Comment:
I would also not bother supplying this value. It should be a developer setup step to generate and configure this key. The app will fail to start without it, and since we're not checking keys into the source it doesn't make sense to provide the path. Or am I missing something?

Reply:
was this question answered?